grails-app/domain/org/pih/warehouse/shipping/ShipmentItem.groovy

```diff
@@ -296,6 +297,14 @@ class ShipmentItem implements Comparable, Serializable {
         return quantityToInvoiceInStandardUom > 0
     }
 
+    OrderItem getOrderItem() {
+        return orderItems[0]
+    }
+
+    String getUom() {
```

Comment:
Looks like your and @drodzewicz are adding the same thing, So it would be to align on the same version (plus naming-wise I would say it should be `getUnitOfMeasure`)